test/albumslist.test.js

```diff
@@ -9,8 +9,10 @@ const request = require('supertest'),
   correctPassword = 'password',
   albumTitle = 'quidem molestiae enim',
   validationErrorStatus = 401,
-  permissionErrorStatus = 403;
-
+  permissionErrorStatus = 403,
+  { notifySignUp } = require('../app/helpers/mailer');
+jest.mock('../app/helpers/mailer');
+notifySignUp.mockResolvedValue(true);
```

Comment:
You are doing this in nearly every file. You could just mock it once in the before each of the app.spec.js.